app/controllers/users_controller.rb

```diff
@@ -1,7 +1,34 @@
 class UsersController < ApplicationController
   def new
+    @user = User.new
   end
 
   def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to edit_user_path
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+
+    if @user.update(user_params)
+      redirect_to new_user_path
```

Comment:
- [ ] This should be redirect_to edit_user_path if the user was saved not new_user_path as the previous TSE mentioned. I think you forgot, please kindly fix it.